sheetwhat/checks/check_funcs.py

```diff
@@ -24,12 +24,15 @@ def check_range(state, field, field_msg, missing_msg=None):
     )
 
 
-def has_code(state, pattern, fixed=False, incorrect_msg=None, normalize=lambda x: x):
+def has_code(state, pattern, fixed=False, index=0, incorrect_msg=None, normalize=lambda x: x):
```

Comment:
Adding an index argument to `has_code` is an interesting idea. We should create a ticket to implement it in other xwhat packages as well then.